src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -673,6 +678,9 @@ ov::AnyMap get_default_generate_config(const std::optional<NPUDesc>& npudesc,
     if (hint == ::intel_npu::npuw::llm::GenerateHint::FAST_COMPILE) {
         config.emplace("NPUW_UNFOLD_IREQS", "YES");
     }
+    // We don't need slice out for kv cache model, especially for speculative decoding which need
+    // to generate more than 1 token for each inference
+    config.erase("NPUW_SLICE_OUT");
```

Comment:
Does it work with `SHARED_LLM_HEAD`? Do we use there `SLICE_OUT` or we have some other patterns?